### DISCUSSION TITLE
1.5.0-beta7

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#390] [Обновления]
> [!WARNING]
> Это бета версия и она может работать нестабильно. Обо всех проблемах рекомендуется сразу же сообщать в [Issues](https://github.com/ilyhalight/voice-over-translation/issues)

## Список изменений
- Теперь, по умолчанию значение опции "слайдер громкости видео" было изменено на включен
- Фикс прогрузки опций при первой инициализации (#389)

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0-beta6...1.5.0-beta7

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0-beta7'>1.5.0-beta7</a>.</em>

**Comment by AvenCores** [comment]
А где собственно билды?

**Reply by ilyhalight** [reply]
Забыл прикрепить, но все равно, всегда, можно скачать билд из прикрепленного архива - `/dist/нужная версия скрипта`

**Comment by CrescentXIII** [comment]
Привет!
На Опере ГХ (99 билд) , появилось странное, когда в меню пытаешься ползунок громкости захватить и потянуть, то будто скриншот начинаешь передвигать (всю картинку, будто стоп-кадр). =)

UPD: прикрепил скрин наглядно
![image](https://github.com/ilyhalight/voice-over-translation/assets/130666598/047b26bc-e56e-4471-b933-215bb9d14f27)

**Reply by ilyhalight** [reply]
> Привет! На Опере ГХ (99 билд) , появилось странное, когда в меню пытаешься ползунок громкости захватить и потянуть, то будто скриншот начинаешь передвигать (всю картинку, будто стоп-кадр). =)
> 
> UPD: прикрепил скрин наглядно ![image](https://user-images.githubusercontent.com/130666598/280558942-047b26bc-e56e-4471-b933-215bb9d14f27.png)

Создайте issue, чтобы я не забыл. Потом, если получится, то пофикшу

upd: это оказалось куда легче пофиксить, чем я предполагал, так, что сейчас сам все оформлю
